extensions/wikia/WikiaRSS/WikiaRssExternalController.class.php

```diff
@@ -1,14 +1,16 @@
 <?php
 class WikiaRssExternalController extends WikiaController {
 
+	const CACHE_TIME = 3600;
```

Comment:
@kvas-damian what value it had before? 
maybe worth asking if 3 hours is good for that use case?